src/EnvChecker.php

```diff
@@ -33,9 +33,7 @@ public function check(string $path): void
             $errors[] = "{$varname} is undocumented";
         }
 
-        if (sizeof($errors)) {
-            throw new InvalidEnvException(implode('; ', $errors));
-        }
+        $this->complain($errors);
```

Comment:
move the throw to its own method where we will first register a special error handler